mne/surface.py

```diff
@@ -2082,3 +2082,73 @@ def _voxel_neighbors(seed, image, thresh=None, max_peak_dist=1,
             next_neighbors = next_neighbors.union(voxel_neighbors)
         neighbors = next_neighbors
     return voxels
+
+
+@verbose
+def project_sensors_onto_surface(info, trans, subject, subjects_dir=None,
```

Comment:
A function that does this shares 95%+ of its functionality like `mne.stc_near_sensors`. These should be refactored to use each other and/or a single shared private function whose job it is to map a sensor to a (usually) `pial` surface location